aikau/src/main/resources/alfresco/renderers/Thumbnail.js

```diff
@@ -611,7 +645,24 @@ define(["dojo/_base/declare",
        * @instance
        */
       getFolderImage: function alfresco_renderers_Thumbnail__getDefaultFolderImage() {
-         return require.toUrl("alfresco/renderers") + "/css/images/" + this.folderImage;
+         var url, jsNode = this.currentItem.jsNode;
+         if(jsNode && jsNode.aspects)
+         {
+            for (var key in this.folderImageAspectMappings)
+            {
+               if(this.folderImageAspectMappings.hasOwnProperty(key) && jsNode.aspects.indexOf("smf:smartFolder") !== -1) 
```

Comment:
This test ... `jsNode.aspects.indexOf("smf:smartFolder") !== -1` only needs to happen once, not on every loop